docs/network/governance.md

```diff
@@ -26,7 +26,7 @@ unilateral fashion.
 
 All parties are held accountable by legal agreements.
 
-## Node Checks and Balances
+## Nodes, Checks and Balances
```

Comment:
If this is the desired title style, it should use a colon `:` instead of a comma `,`. (Same all the way down for the test of the titles).

Reply:
+1, thanks.

Reply:
Oh, I just read your comment. Hmm, I need to rethink this, it's messier than it needs to be..

Reply:
Ok, well, we can leave it with the colon for now then! :)

Reply:
@creatinglake -- I've given my input. -- you make the call. Don't water it down or overcomplicate it on my account.